### ISSUE TITLE
max_messages ?

### ISSUE THREAD

**Issue by ScottBrenner** [logstash-plugins/logstash-input-google_pubsub#13]
The [`max_messages`](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-google_pubsub.html#plugins-inputs-google_pubsub-max_messages) configuration option is missing a description, what does it do? It's allegedly required, although I'm able to run Logstash without setting it.

**Comment by amrmotan** [comment]
I agree, since this is an important parameter, there should be a clear documentation on how to use it.

**Comment by amrmotan** [comment]
I think this issue can be closed now since the description was added.
